components/automate-ui/src/app/modules/team/team-add-users/team-add-users.component.html

```diff
@@ -10,8 +10,10 @@
   (confirm)="addUsers()"
   (close)="closePage()">
   <div *ngIf="!(loading$ | async)">
+    <app-create-user-modal [openEvent]="openUserModal"></app-create-user-modal>
     <div id="no-users-container" *ngIf="usersNotFiltered().length === 0">
-      <p>There are no more local <a [routerLink]="['/settings', 'users']" target="_blank">users</a> to add; create some more!</p>
+      <p>There are no more local users to add; create some more!</p>
+      <chef-button primary (click)="openModal()">Create User</chef-button>
```

Comment:
These are the only template changes necessary on the "Add Users" dialog for teams: the modal and the button.